_posts/2020-07-22-Solidity-0612-release-announcement.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.6.12 Release Announcement'
+date: '2020-07-22'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.6.12](https://github.com/ethereum/solidity/releases/tag/v0.6.12) adds more flexibility
+when using inheritance with NatSpec comments and introduces many small improvements to the optimizer.
+
+
+## Notable New Features 
+
+### NatSpec Inheritance and on Events
```

Comment:
I don't understand this title. Is "and on" correct here?

Reply:
I think it's correct but "and on Events" should be removed.

Reply:
done!